src/pages/NewsList/NewsList.tsx

```diff
@@ -127,51 +127,63 @@ export default function NewsList() {
 									<a
 										href={`${routes.newsItem}?id=${newsItem.time_published}`}
 										onClick={handleClick(newsItem)}
+										className="before:absolute before:inset-0 hover:underline hover:decoration-gray-500"
```

Comment:
it looks like you have the same code twice in your codebase, here and in HP, maybe add a component?